packages/bundler/src/modules/BundleManager.ts

```diff
@@ -44,8 +51,10 @@ export class BundleManager {
     // in conditionalRpc: always put root hash (not specific storage slots) for "sender" entries
     readonly mergeToAccountRootHash: boolean = false
   ) {
-    this.provider = entryPoint.provider as JsonRpcProvider
-    this.signer = entryPoint.signer as JsonRpcSigner
+    // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+    this.entryPoint = _entryPoint!
+    this.provider = this.entryPoint.provider as JsonRpcProvider
```

Comment:
if entrypoint is null, then should pass provider explicitly, otherwise, this would crash.